test/controllers/ReviewPenaltyController.test.ts

```diff
@@ -0,0 +1,84 @@
+import 'reflect-metadata';
+
+import { Penalty, PenaltyList } from 'ch-sdk-node/dist/services/lfp';
+import { expect } from 'chai';
+import { OK } from 'http-status-codes';
+import request from 'supertest';
+
+import 'app/controllers/ReviewPenaltyController';
+import { Appeal } from 'app/models/Appeal';
+import { OTHER_REASON_DISCLAIMER_PAGE_URI, REVIEW_PENALTY_PAGE_URI } from 'app/utils/Paths';
+
+import { createApp } from 'test/ApplicationFactory';
+
+describe('ReviewPenaltyController', () => {
+
+    const companyNumber: string = 'NI000000';
+    const companyName: string = 'Test Ltd.';
+    const penaltyReference: string = 'A00000000';
```

Comment:
I know this is just testing, but that penalty reference has 9 characters, which doesn't match the modern penalty reference number format. I think we should change it to A0000000 (8 characters)